app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end 
+
+  # Create --------------------------------------------------------------------
+
+  get '/tweets/new' do
+    logged_in? ? (erb :'tweets/create_tweet') : (redirect to '/login')
+  end 
+
+  post '/tweets' do
+    if params[:content].empty?
+      flash[:message] ="Provide some content to post a tweet!"
+      redirect to '/tweets/new'
+    elsif logged_in?
+      @user = current_user
+      @tweet =Tweet.create(params)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.